### ISSUE TITLE
RTL and Locale

### ISSUE THREAD

**Issue by israelidanny** [facebook/fixed-data-table#417]
I noticed that everything is almost fully ready for good RTL - there are just a couple of `left: 0;` to be adapted in the CSS and `Locale.js` needs to not be hardcoded.

What kind of PR would be accepted to further support of RTL (based on body tag rtl for example?)

**Comment by wcjordan** [comment]
Hey @israelidanny the maintainers of this repo are no longer active, but a few of us are actively working on a fork here: https://github.com/schrodinger/fixed-data-table-2

If you'd like to open an issue or submit a PR there, we'd be happy to take a look.  Our guide for contributing can be found here: https://github.com/schrodinger/fixed-data-table-2/blob/master/CONTRIBUTING.md

**Comment by Daniel15** [comment]
> there are just a couple of left: 0; to be adapted in the CSS

Good catch. At Facebook we have a transform that automatically converts things like `left: 0` to `right: 0` and `margin-left` to `margin-right` for RTL languages. This means we generally don't have to worry about having separate CSS files for our internal use cases. Probably worth having separate RTL stylesheets for the open-source release, or a similar transform that generates them.
